src/components/Book.js

```diff
@@ -1,28 +1,28 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { deleteBook } from '../redux/Books/Books';
 
-const Book = ({ books, propsToRemoveBooks }) => (
-  <ul>
-    {books.map((book) => (
-      <li key={book.id}>
-        <span>{book.title}</span>
-        {' '}
-        <br />
-        <span>by</span>
-        {' '}
-        <br />
-        <span>{book.author}</span>
-        {' '}
-        <br />
-        <button type="button" onClick={() => propsToRemoveBooks(book.id)}>Remove</button>
-      </li>
-    ))}
-  </ul>
-);
+const Book = ({ books }) => {
+  const dispatch = useDispatch();
+  return (
+    <ul>
+      {books.map((book) => (
+        <li key={uuidv4()}>
+          <span>{book.title}</span>
+          <br />
+          <span>{book.category}</span>
+          <br />
+          <button type="button" onClick={() => dispatch(deleteBook(book.id))}>Remove</button>
```

Comment:
[OPTIONAL] It would be good if you deconstruct `book` -> `{ title, category, id }`. This way you avoid repeating code and it's a Best Practice on JS.